packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -165,8 +186,13 @@ export default class BotonicPluginFlowBuilder implements Plugin {
     }
     return result.target.id
   }
+
+  getPayloadParams<T>(payload: string): T & PayloadParamsBase {
```

Comment:
I think this function should be placed in another file like utils or something like that.

Reply:
I need to export this function from the plugin's index to be able to do the import in a simple way as

`import { PayloadParamsBase } from '@botonic/plugin-flow-builder'`

Reply:
I want it inside the plugin for you to use by doing `cmsPlugin.getPaylodParams<PayloadParamsBase>(request.input.payload)` as seen in the description of this PR. It can be another type extending from PayloadParamsBase.

Reply:
To me, makes sense that the methods declared in a class do operations with elements of the class. In my opinion and following the SRP this method is independent from the class and should be placed into a utils file and be exported from there. But this is an opinion, if you think it makes more sense having it there, keep it.